{
	"name": "try_except",
	"module": "pyPythonRPA.Robot.pythonRPA",
	"class": "errors",
	"function": "try_except",
	"class_params": {},
	"func_params": {
		"try_case": {
			"type": "string",
			"example": "ex:'next_id_in_try'",
			"required": true,
			"path": false,
			"output": true
		},
		"except_case": {
			"type": "string",
			"example": "ex:'next_id_in_except'",
			"required": true,
			"path": false,
			"output": true
		},
		"after": {
			"type": "string",
			"example": "ex:'next_id'",
			"required": true,
			"path": false,
			"output": true
		},
		"state": {
			"initial_state": true
		}
	},
	"results": null,
	"object": null,
	"inherit_object": null,
	"breakpoint": false,
	"if_error": {
		"go_to": "block_id",
		"repeat": 1,
		"counter": 0,
		"delay": 0
	},
	"start": false,
	"next_id": "condition",
	"additional_info": {
		"inheritance": false
	}
}